### DISCUSSION TITLE
Is there a Github Desktop MacOS ARM installer?

### DISCUSSION THREAD

**Discussion by cosmasterx** [community/community#120049] [New to GitHub]
Hello,

I am looking for an installer for macOS - ARM. Is there one? If so, where can I find it?

**Comment by P3tKa** [comment]
Hey,

Yes there is an **macOS ARM** installer for **Github Desktop**. You can **download** it by clicking this [link](https://central.github.com/deployments/desktop/desktop/latest/darwin-arm64) or by following instructions below:

Visit [https://desktop.github.com](https://desktop.github.com) and click on `Download for an Apple silicon`

![screenshot](https://github.com/community/community/assets/64527557/317f1d1d-f38e-493c-b5be-6959999866f2)
